core-services/prow/02_config/_plugins.yaml

```diff
@@ -607,6 +607,11 @@ config_updater:
         app.ci:
         - ci
       name: ci-operator-misc-configs
+    ci-operator/config/integr8ly/integreatly-operator/integr8ly-integreatly-operator-*-next-*.yaml:
+      clusters:
+        app.ci:
+        - ci
+      name: ci-operator-misc-configs
```

Comment:
I tried to do this before for our release branches, and then after discussion, ended up changing the branch names we used to match an already existing config. 
@stevekuznetsov Do you see any issues with this one? This should only ever match on our repo this way right? 

Branch names we want to match are like `amq-online-next-1.5.0`, `3scale-next-0.6.0`, if needs be i can change these since we generate them.

Reply:
As long as it does not make a branches match  more than one pattern, this is ok!